components/Header.js

```diff
@@ -28,6 +41,9 @@ const StyledHeader = styled.section`
   border-bottom-right-radius: 10px;
   border-bottom-left-radius: 10px;
   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  display: flex;
```

Comment:
uhhhh. z-index in line 40 🙈 
maybe you can delete line 38 and 39 as it is not in use. Also you may want to think about using var(--color-name) for your colors. Just as a tipp

Reply:
The code in 38, 39 is now in use again, i need the code and the  zindex to get the header over the map :/ 
Where do i have to set the color vars?

Reply:
you can set it in :root in your globalStyles